src/CLI_Installer/Product.wxs

```diff
@@ -59,36 +59,36 @@ Licensed under the MIT license. See LICENSE file in the project root for full li
 
     <ComponentGroup Id="ProductComponents" Directory="INSTALLFOLDER">
       <Component Id="ProductComponent" Guid="548A9965-21AD-4CDF-99AD-A6F2A47D0AE8">
-        <File Source="..\CLI\bin\Release\netcoreapp3.1\AxeWindowsCLI.exe" />
-        <File Source="..\CLI\bin\Release\netcoreapp3.1\AxeWindowsCLI.dll" />
-        <File Source="..\CLI\bin\Release\netcoreapp3.1\AxeWindowsCLI.deps.json" />
-        <File Source="..\CLI\bin\Release\netcoreapp3.1\AxeWindowsCLI.runtimeconfig.json" />
-        <File Source="..\CLI\bin\Release\netcoreapp3.1\Axe.Windows.Actions.dll" />
-        <File Source="..\CLI\bin\Release\netcoreapp3.1\Axe.Windows.Automation.dll" />
-        <File Source="..\CLI\bin\Release\netcoreapp3.1\Axe.Windows.Core.dll" />
-        <File Source="..\CLI\bin\Release\netcoreapp3.1\Axe.Windows.Desktop.dll" />
-        <File Source="..\CLI\bin\Release\netcoreapp3.1\Axe.Windows.Rules.dll" />
-        <File Source="..\CLI\bin\Release\netcoreapp3.1\Axe.Windows.RuleSelection.dll" />
-        <File Source="..\CLI\bin\Release\netcoreapp3.1\Axe.Windows.SystemAbstractions.dll" />
-        <File Source="..\CLI\bin\Release\netcoreapp3.1\Axe.Windows.Telemetry.dll" />
-        <File Source="..\CLI\bin\Release\netcoreapp3.1\Axe.Windows.Win32.dll" />
-        <File Source="..\CLI\bin\Release\netcoreapp3.1\CommandLine.dll" />
-        <File Source="..\CLI\bin\Release\netcoreapp3.1\Microsoft.Win32.SystemEvents.dll" />
-        <File Source="..\CLI\bin\Release\netcoreapp3.1\Newtonsoft.Json.dll" />
-        <File Source="..\CLI\bin\Release\netcoreapp3.1\System.Drawing.Common.dll" />
-        <File Source="..\CLI\bin\Release\netcoreapp3.1\System.IO.Packaging.dll" />
-        <File Source="..\CLI\bin\Release\netcoreapp3.1\thirdpartynotices.html" />
-        <File Source="..\CLI\bin\Release\netcoreapp3.1\README.MD" />
+        <File Source="..\CLI_Full\bin\Release\netcoreapp3.1\win7-x86\AxeWindowsCLI.exe" />
```

Comment:
This may be a problem. IIRC, the EXE from the CLI_full project is stamped in a way that prevents it from loading any of the locally-installed .NET assemblies. If that's the case, then the MSI-installed CLI will be broken as soon as it needs to use anything that isn't installed in the MSI. Easy to check--install the MSI and see if the CLI can still function as expected.

Reply:
Reverted Product.wxs to original state, moved 3rd party signing to CLI, and updated zip creation script to source from CLI and CLI_Full as appropriate.